src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -482,6 +482,26 @@ public override Uri GetPackageAbsoluteUri(PackageURL purl)
             JsonDocument jsonDoc = await GetJsonCache(client, $"{ENV_NPM_API_ENDPOINT}/{packageName}", useCache);
             return ParseUploadTime(jsonDoc, purl.Version);
         }
+        public override async Task<DateTime?> GetPackageCreatedAtUtcAsync(PackageURL purl, bool useCache = true)
+        {
+            HttpClient client = CreateHttpClient();
+            string? packageName = purl.HasNamespace() ? $"{purl.GetNamespaceFormatted()}/{purl.Name}" : purl.Name;
+            JsonDocument jsonDoc = await GetJsonCache(client, $"{ENV_NPM_API_ENDPOINT}/{packageName}", useCache);
+            return ParseCreatedTime(jsonDoc);
+        }
```

Comment:
It's not something we need to address in this particular PR, but the current separate-method-for-each-separate-metadata-field approach feels a bit clunky and might involve redundant JSON parsing depending on how efficient `GetJsonCache` is.

There are always tradeoffs - but at least for npm where we have to parse a bunch of stuff to get a single property anyway we might as well put all of the stuff we parsed into a single 'metadata' object in one go and return that one object (instead of making separate NPMProjectManager method calls for every individual thing we might want)

Reply:
I agree it's different conceptually than this PR, but agree that a structured metadata object would be preferable to adding new methods for each field.

Reply:
I have removed the separate method to fetch the individual metadata fields in favor of using GetPackageMetadataAsync method to fetch metadata.